lib/internal/bootstrap/pre_execution.js

```diff
@@ -30,6 +35,7 @@ function prepareMainThreadExecution(expandArgv1 = false) {
   setupPerfHooks();
   setupInspectorHooks();
   setupWarningHandler();
+  setupFetch();
```

Comment:
It seems that this is not the right place to run this code if I want `fetch` to also be installed in workers.
Note that I cannot do it in `internal/bootstrap/browser.js` because it's not allowed to call `getOptionValue` from this file.

Reply:
I think it is fine leaving it here for now, we can surely move it to `browser.js` after the experimental flag is removed. Should we add a code comment to remind about this?